### PR TITLE
TCM-105 | Build dummy content screen 

### PR DESCRIPTION
**Pull request by halilyesilyurt**
### Summary

- [x] Completed dummy content screen.
- [x] Added dummy content to test utility.

### Issue

[TCM-105](https://hyfnla.atlassian.net/browse/TCM-105)

### Video/Screenshot

| State Book| State Plain|
|--------|--------|
|content_state_book|content_state_plain|

### CODE REVIEW

File: app/src/main/java/com/trx/consumer/models/common/WaiverModel.kt [hunk 1]
```diff
@@ -0,0 +1,17 @@
+package com.trx.consumer.models.common
+
+import android.os.Parcelable
+import kotlinx.parcelize.Parcelize
+import org.json.JSONObject
+
+@Parcelize
+data class WaiverModel(var covid: String = "", var terms: String = "") : Parcelable {
```
Comment (VivekV95) [inline app/src/main/java/com/trx/consumer/models/common/WaiverModel.kt:8]: Think we can remove this as iOS doesn't have it

File: app/src/main/java/com/trx/consumer/models/common/ContentModel.kt [hunk 2]
```diff
@@ -0,0 +1,13 @@
+package com.trx.consumer.models.common
+
+import android.os.Parcelable
+import com.trx.consumer.screens.content.ContentViewState
+import kotlinx.parcelize.Parcelize
+
+@Parcelize
+class ContentModel(
+    var state: ContentViewState = ContentViewState.PLAIN,
```
Comment (VivekV95) [inline app/src/main/java/com/trx/consumer/models/common/ContentModel.kt:9]: Let's remove the `state` and `waiverModel` properties to match iOS

File: app/src/main/java/com/trx/consumer/screens/content/ContentViewState.kt [hunk 3]
```diff
@@ -0,0 +1,5 @@
+package com.trx.consumer.screens.content
+
+enum class ContentViewState {
```
Comment (VivekV95) [inline app/src/main/java/com/trx/consumer/screens/content/ContentViewState.kt:3]: Think we can remove this as iOS doesn't have it

**Review by VivekV95 — CHANGES_REQUESTED** [review]
Few things we can remove that I don't think we'll need in TRX

**Review by onodude — APPROVED** [review]

**Pull request merged**